.mci.yml

```diff
@@ -3,6 +3,17 @@
 #######################################
 
 #######################################
+#            Variables                #
+#######################################
+
+linux_cmake_path: &linux_cmake_path /opt/cmake/bin/cmake
```

Comment:
I feel like these will expand in the future, so can we divide them up a bit further?  I'm thinking either by operating system or by feature (like "cmake path").  Also, just for good form, let's put these in some kind of bigger "variables" structure.  It's possible that one day evergreen will be able to parse something like that in a helpful way.

```
## variables
variables: 

     ## linux variables
     linux:
          linux_cmake_path: &name stuff
          linux_source: &name stuff
          ...
     ## osx variables
     osx:
          osx_cmake_path: &name stuff
          ...
```